.mergify.yml

```diff
@@ -14,3 +14,16 @@ pull_request_rules:
         method: squash
         strict: smart
         strict_method: merge
+
+  - name: Automatically approve Dependabot PRs
```

Comment:
Does this auto-approve, or auto-merge? Won't we still need to approve the PRs before they get mergified?

Reply:
NVM, I understand this now.